src/Toggle/Toggle.js

```diff
@@ -66,6 +76,32 @@ function getStyles(props, context, state) {
     },
   };
 
+  Object.assign(styles.track,
+    trackStyle,
+    state.switched && styles.trackWhenSwitched,
+    state.switched && trackSwitchedStyle,
+    disabled && styles.trackWhenDisabled
+  );
+
+  Object.assign(styles.thumb,
+    thumbStyle,
+    state.switched && styles.thumbWhenSwitched,
+    state.switched && thumbSwitchedStyle,
+    disabled && styles.thumbWhenDisabled
+  );
+
+  if (state.switched) {
+    styles.thumb.marginLeft = 0 - styles.thumb.width;
+  }
+
+  Object.assign(styles.icon, iconStyle);
```

Comment:
In the other components, we are handling the overiding style by users in the `render` method.
Sorry for suggesting you going into this direction.
Ultimatly, those components have been rewrote on the `next`  branch. That shouldn't matter much.